ui/src/pages/author-guide.tsx

```diff
@@ -50,332 +334,30 @@ const AuthorGuide: NextPage = (): React.ReactElement => (
-                                    has agreed it, the work will be live and a DOI available.
-                                </li>
-                                <li className="mb-6">
-                                    <span className="font-bold">What if I made a mistake?</span> Octopus will allow for
-                                    reversioning of publications, but this feature is currently under development. When
-                                    released, reversioning is intended for cases where an author makes a mistake, or a
-                                    reviewer points out something which needs to be added or corrected. In the latter
-                                    case, we strongly recommend including the reviewer as a co-author on a new version
-                                    to recognise their input. All older versions remain visible on the record, but new
-                                    versions have their own DOI. All reviews and red flags remain with the version they
-                                    were originally linked to. We recommend keeping reversioning to a minimum.
-                                </li>
-                            </ul>
-                        </StandardText>
-                        <StandardText>
-                            <>
-                                If you require further guidance, or would like to suggest any additional author
-                                materials, please contact the team at{' '}
-                                <Components.Link
-                                    href="mailto:help@jisc.ac.uk"
-                                    openNew={true}
-                                    className="rounded text-teal-600 outline-0 transition-colors duration-500 focus:ring-2 focus:ring-yellow-400 dark:text-teal-300"
-                                >
-                                    help@jisc.ac.uk.
-                                </Components.Link>
-                            </>
-                        </StandardText>
-                    </div>
-                </>
-            </PageSection>
+            <div className="container mx-auto px-8 pb-10 pt-10 lg:gap-4 lg:pt-20">
+                <Components.PageTitle text="Author guide to publishing on Octopus." />
+            </div>
+            <div className="container mx-auto grid grid-cols-1 px-8 lg:grid-cols-8 lg:gap-16">
+                <aside className="col-span-2 pt-2 lg:block">
+                    <Components.PageContentsSidebar
+                        jumpToList={authorGuideSections.map(({ title, id }) => ({ href: id, title }))}
+                    />
+                </aside>
+                <div className="pt-14 lg:col-span-6 lg:pt-0">
+                    {authorGuideSections.map((authorGuideSection) => (
+                        <div key={authorGuideSection.id} id={authorGuideSection.id} className="mx-auto pt-4 lg:w-10/12">
```

Comment:
Could drop this div and apply styling, key, id etc to the `<dl />`

Reply:
This issue should be gone with the new structure I've just written